src/views/v2/month/calendar-body/day/calendar-events/calendar-event/tooltip/description.php

```diff
@@ -16,11 +16,10 @@
  * @see tribe_get_event() For the format of the event object.
  */
 
-/**
- * @todo: @be @bordoni
- *        do not output html if excerpt is empty
- */
+if ( empty( $event->excerpt ) ) {
+	return;
+}
 ?>
 <div class="tribe-events-calendar-month__calendar-event-tooltip-description tribe-common-b3">
-	<?php echo tribe_events_get_the_excerpt( $event->ID, wp_kses_allowed_html( 'post' ) ); ?>
+	<?php echo $event->excerpt; ?>
```

Comment:
should this have `esc_html()`?

Reply:
No, the `excerpt` is generated with a template tag that takes care of escaping, and we're actually allowing a bit of HTML. 

- https://github.com/moderntribe/the-events-calendar/pull/2727/files#diff-099cdbe9ae7ffadddaa170ce58e565fbR153
- https://github.com/moderntribe/the-events-calendar/blob/release/B19.13/src/functions/template-tags/general.php#L1450